src/cli/api_client.rs

```diff
@@ -368,6 +368,62 @@ impl ApiClient {
         );
         Ok(())
     }
+
+    #[allow(clippy::too_many_arguments)]
+    pub async fn update_batch_group(
+        &self,
+        id: String,
+        description: Option<String>,
+        tx_priority: Option<TxPriority>,
+        consolidate_deprecated_keychains: Option<bool>,
+        manual_trigger: Option<bool>,
+        immediate_trigger: Option<bool>,
+        interval_trigger: Option<u32>,
+    ) -> anyhow::Result<()> {
+        let tx_priority = tx_priority.map(|priority| match priority {
+            TxPriority::NextBlock => proto::TxPriority::NextBlock as i32,
+            TxPriority::OneHour => proto::TxPriority::OneHour as i32,
+            TxPriority::Economy => proto::TxPriority::Economy as i32,
+        });
+
+        let trigger = if manual_trigger.is_some() {
```

Comment:
use `if let Some(manual_trigger) = manual_trigger`
then you don't have to call unwrap.